schema/schema.yaml

```diff
@@ -16,7 +16,8 @@ tables:
       datatype: string
       length: 30
       description: Publication reference identifier
-      ivoa:ucd: meta.id;meta.main
+      ivoa:ucd: meta.ref;meta.main
+      nullable: false
```

Comment:
We might not need an explicit `nullable: false` since it's a primary key, but it doesn't hurt to have